### PR TITLE
Document Lazy decorator.

### PR DESCRIPTION
**Pull request by DanielSank**

### CODE REVIEW

File: labrad/units.py [hunk 1]
```diff
@@ -129,15 +129,46 @@ def __div__(self, other):
...
+            return intense_computation(self, ...)
+
+    >>> foo = Foo()
+    >>> result_first_access = foo.bar
+    # Incurs long computation. Note the lack of ().
+    
+    >>> result_second_access = foo.bar
+    # Incurs no computation. Looks up previously computed and cached result.
+
+    Methods decorated by Lazy mus take no arguments aside from the implicit self
```
Comment (maffoo) [inline labrad/units.py]: typo: "must"

Comment (jwenner) [inline labrad/units.py]: I think that was a mus-take.

Comment (ejeffrey) [inline labrad/units.py]: I think when @jwenner makes a pun he should be auto-assigned the bug :)

Comment (DanielSank) [inline labrad/units.py]: +1

**Comment by DanielSank** [comment]
@maffoo fixed

**Comment by maffoo** [comment]
thanks for the change! merged to `u/maffoo/optimize-units`

**Pull request closed without merging**